src/main/java/org/openrewrite/java/migrate/AddScopeToInjectedClass.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.migrate;
+
+import lombok.Data;
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.ScanningRecipe;
+import org.openrewrite.Tree;
+import org.openrewrite.TreeVisitor;
+import org.openrewrite.java.JavaIsoVisitor;
+import org.openrewrite.java.AnnotationMatcher;
+import org.openrewrite.java.JavaTemplate;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.internal.lang.Nullable;
+import org.openrewrite.java.tree.*;
+
+import java.util.*;
+
+public class AddScopeToInjectedClass extends ScanningRecipe<AddScopeToInjectedClass.Accumulator> {
+    private static final Collection<String> typesPromptingScopeAddition = Arrays.asList("javax.inject.Inject");
+
+    @Override
+    public String getDisplayName() {
+        return "Add scope annotation to injected classes";
+    }
+
+    @Override
+    public String getDescription() {
+        return "Add scope annotation to injected classes.";
```

Comment:
We usually expand a bit in the description to tell what changes will be made, and can use markdown here to for instance call out the `@Dependent` and `@Inject` annotations within the String.